### PR TITLE
add embedded behavior for mapbuttoncontainer

### PR DESCRIPTION
**Pull request by alsturm**

### CODE REVIEW

**Review by taulinger — COMMENTED** [review]
Please don't use `falsy` / `truthy` anymore.
(https://vincenttunru.com/toBeTruthy-vs-toBe-true/)

Let's use  `querySelectorAll` and `toHaveSize` instead

File: test/modules/map/components/mapButtonsContainer/MapButtonsContainer.test.js [hunk 1]
```diff
@@ -5,17 +5,21 @@ import { $injector } from '../../../../../src/injection';
...
 describe('MapButtonsContainer', () => {
-	const setup = (state = {}) => {
+	const setup = (state = {}, config = {}) => {
+		const { embed = false } = config;
 		const initialState = {
 			media: {
 				portrait: false,
-				minWidth: true
+				minWidth: true,
+				embed: false
```
Comment (taulinger) [inline test/modules/map/components/mapButtonsContainer/MapButtonsContainer.test.js:14]: Property `embed` is not a part of the slice-of-state `media`

File: test/modules/map/components/mapButtonsContainer/MapButtonsContainer.test.js [hunk 2]
```diff
@@ -42,10 +46,10 @@ describe('MapButtonsContainer', () => {
...
 			const element = await setup(state);
 
-			expect(element.shadowRoot.querySelector('.is-portrait')).toBeFalsy();
-			expect(element.shadowRoot.querySelector('.is-landscape')).toBeTruthy();
-			expect(element.shadowRoot.querySelector('.is-desktop')).toBeTruthy();
-			expect(element.shadowRoot.querySelector('.is-tablet')).toBeFalsy();
+			expect(element.shadowRoot.querySelectorAll('.is-portrait')).toHaveSize(0);
+			expect(element.shadowRoot.querySelectorAll('.is-landscape')).toHaveSize(1);
+			expect(element.shadowRoot.querySelectorAll('.is-desktop')).toHaveSize(1);
+			expect(element.shadowRoot.querySelectorAll('.is-tablet')).toHaveSize(0);
```
Comment (taulinger) [inline test/modules/map/components/mapButtonsContainer/MapButtonsContainer.test.js:52]: Thx 👍

File: test/modules/map/components/mapButtonsContainer/MapButtonsContainer.test.js [hunk 3]
```diff
@@ -99,13 +103,31 @@ describe('MapButtonsContainer', () => {
...
+			expect(element.shadowRoot.querySelectorAll('.is-desktop')).toHaveSize(0);
+			expect(element.shadowRoot.querySelectorAll('.is-tablet')).toHaveSize(1);
 
 			expect(window.getComputedStyle(element.shadowRoot.querySelector('ba-zoom-buttons')).display).toBe('none');
 			expect(window.getComputedStyle(element.shadowRoot.querySelector('ba-extent-button')).display).toBe('none');
 		});
 	});
+
+	describe('embedded layout ', () => {
+		it('layouts for not embedded', async () => {
```
Comment (taulinger) [inline test/modules/map/components/mapButtonsContainer/MapButtonsContainer.test.js:117]: "layouts for default mode"

File: test/modules/map/components/mapButtonsContainer/MapButtonsContainer.test.js [hunk 4]
```diff
@@ -99,13 +103,31 @@ describe('MapButtonsContainer', () => {
...
+			const element = await setup({}, { embed: false });
+
+			expect(element.shadowRoot.querySelectorAll('.is-embedded')).toHaveSize(0);
+		});
+
+		it('layouts for embedded', async () => {
```
Comment (taulinger) [inline test/modules/map/components/mapButtonsContainer/MapButtonsContainer.test.js:123]: "layouts for embedded mode"

Comment (alsturm) [reply test/modules/map/components/mapButtonsContainer/MapButtonsContainer.test.js]: done

**Review by taulinger — APPROVED** [review]
Nice!

**Pull request merged**